src/components/inputs.js

```diff
@@ -1,25 +1,73 @@
 import './input.css';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux/es/exports';
+import { addBook } from './redux/books/books';
 
-const Input = () => (
-  <div className="inputs">
-    <div>
-      <li>
-        <input type="text" className="book-title" placeholder="Book title" />
-      </li>
-    </div>
-    <div>
-      <li>
-        <select id="books" name="books" placeholder="categories">
-          <option value="categories">categories</option>
-        </select>
-      </li>
-    </div>
+const Input = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+
+  const handleTitle = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const handleAuthor = (e) => {
+    setAuthor(e.target.value);
+  };
+
+  const handleSubmit = () => {
+    const mybook = {
+      id: Math.floor(Math.random() * 100),
+      title,
+      author,
+    };
```

Comment:
- [ ] Kindly make sure the API is working. the is an error occurring when submitting this form saying the category should not be empty.

![image](https://user-images.githubusercontent.com/65068771/179373707-e62b4aac-6757-410b-ad73-9b8a50d3a463.png)